packages/fraction-model/controller/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@pie-element/fraction-model-controller",
+  "private": true,
+  "version": "2.17.1",
```

Comment:
we should use an initial version like 1.0.0

Reply:
Updated